crates/noirc_evaluator/src/ssa_refactor/ir/printer.rs

```diff
@@ -87,7 +97,9 @@ pub(crate) fn display_terminator(
             writeln!(
                 f,
                 "    jmpif {} then: {}, else: {}",
-                condition, then_destination, else_destination
+                value(function, *condition),
```

Comment:
Some small printing improvements as well. The jmpif condition value wasn't being displayed properly if it was a constant before (need to call the value display function), and I changed block parameters as well to print out their types.